README.md

```diff
@@ -150,17 +150,16 @@ The code in this repo aims to be compatible with:
 All the code should mainly follow the [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml).
 In particular:
 
-* The first line of every script should be `#!/bin/bash`.
+* The first line of every script should be `#!/usr/bin/env bash`.
 * All code should be defined in functions.
 * Functions should exit or return 0 on success and non-zero on error.
 * Functions should return output by writing it to `stdout`.
 * Functions should log to `stderr`.
 * All variables should be `local`. No global variables are allowed at all.
 * Make as many variables `readonly` as possible.
```

Comment:
Ah nice, I should have actually left that there. Mind if I also add a bunch references to explain why we intend to enforce these rules? It will help users understand the 'why' and also help them learn some Bash.